hydra-cluster/src/Hydra/Cluster/Faucet.hs

```diff
@@ -43,7 +42,7 @@ seedFromFaucet ::
   -- | Marked as fuel or normal output?
   Marked ->
   IO UTxO
-seedFromFaucet networkId (RunningNode _ nodeSocket) receivingVerificationKey lovelace marked = do
+seedFromFaucet RunningNode{networkId, nodeSocket} receivingVerificationKey lovelace marked = do
```

Comment:
Ah! This is where the `networkId` has gone! Great, yes indeed, we talked about it.